staging/src/k8s.io/cloud-provider/fake/fake.go

```diff
@@ -86,6 +86,7 @@ type Cloud struct {
 	ExternalIP    net.IP
 	Balancers     map[string]Balancer
 	UpdateCalls   []UpdateBalancerCall
+	UpdateCallCh  chan UpdateBalancerCall
```

Comment:
this is not great for a general mock, also it will depend on the users to create the channel and buffered or it will block, can we find a better way than this?

Reply:
I couldn't at least. We need a way to be able to trigger an update on the service while the node sync is processing, besides using a channel I didn't have any other ideas.

Reply:
no ideas sorry,